src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -14180,6 +14356,41 @@ Label900: ;
 		return StringOut;
 	}
 
+	bool 
+	isNumber(std::string s)
+	{
+		bool is = true;
+		try {
+			std::stof(s);
+		} catch ( std::exception & e ) {
```

Comment:
I thought it took a third argument to specific the base of the number, hence the 10. Looking at the documentation, it doesn't seem to take a third parameter like you said and instead automatically determines the base. You should just remove that last input parameter. I did say it was untested 😉